src/components/Dialog.tsx

```diff
@@ -67,37 +64,46 @@ export function Provider({ value, children }: ProviderProps) {
 const OnCloseContext = createContext<(() => void) | undefined>(undefined)
 
 const HeaderRow = styled(Row)`
```

Comment:
should this be

```suggestion
const HeaderRow = styled(Row)<{iconSize?: number}>`
```

? i am new to styled components, so i'm not sure if we should always use the prop types